### DISCUSSION TITLE
Compiling Jyers

### DISCUSSION THREAD

**Discussion by sylvain-girard** [Jyers/Marlin#1466] [Q&A] [answered]
I'm not quite sure what I'm doing wrong here, I'm trying to compile the firmware myself so I can increase the bed size and adjust my X and Y offsets, but after compiling and refreshing, it seems none of the changes worked. The firmware definitely changed to Jyers, however the bed size remained at 220x220 with no levelling function. However using the pre-compiled version works as expected. I have compiled the stock firmware before without any issues. The way I'm doing it now is just downloading the source code, opening it through VS code, finding the template I need, adjust, compile, and reflash.

**Comment by Jyers** [comment, marked as answer]
That should be all you need, perhaps the firmware isn't actually flashing? Maybe try reformatting the SD card.
